test/jdk/tools/jpackage/junit/share/jdk.jpackage/jdk/jpackage/internal/AppImageFileTest.java

```diff
@@ -117,7 +117,7 @@ AppImageFile create() {
                     List.of(), false, null, Optional.empty(), null, Map.of());
 
             final var app = new Application.Stub(null, null, version, null, null,
-                    Optional.empty(), List.of(), null, Optional.empty(),
+                    Optional.empty(), Optional.empty(), List.of(), null, Optional.empty(),
```

Comment:
Why is this change? The `Application` interface has not changed. How does it compile? Am I missing something?